infra/modules/providers/azure/service-plan/main.tf

```diff
@@ -0,0 +1,27 @@
+resource "random_id" "remotestate_account_name" {
+  byte_length = 6
+  keepers {
+    sa_account_ref = 1
+  }
+}
+
+resource "azurerm_resource_group" "svcplan" {
+  name     = "${var.resource_group_name == "" ? "${local.name}-cobalt-rg" : "${var.resource_group_name}"}"
```

Comment:
`name` should be driven directly off `resource_group_name `

Reply:
Check response as above.